util.go

```diff
@@ -67,8 +62,22 @@ func getService(c *cli.Context) (*route53.Route53, error) {
 	config, err := getConfig(c)
 	if err != nil {
 		return nil, err
-	}
-	return route53.New(session.New(), config), nil
+  }
+  profile := c.String("profile")
+  if profile != "" {
+    sess := session.Must(session.NewSessionWithOptions(session.Options{
```

Comment:
`session.Must` isn't right here - it forces a panic on error. `getService` should return the error so it can be presented to the user in a friendly manner.